index.html

```diff
@@ -767,24 +767,68 @@ <h1>Streaming GCN<br>Notices in Python</h1>
     <section>
       <div class="image-and-caption">
         <div>
-          <img src="img/screen-shot-gcn-markdown.png" width="700" alt="Screen shot of GCN Circular web form">
+          <img src="img/screen-shot-circulars.png" width="700" alt="Screen shot of GCN Circulars archive">
         </div>
         <div data-markdown>
           <textarea data-template>
-            ## Modernization of GCN Circulars (Coming Soon!)
-            * Manage your own subscriptions
-            * Self-register to submit using peer endorsement
-            * Submit circulars by web form, API, or email
-            * New searchable archive of Circulars
-            * Planned enhancements
-              * "Astro-Flavored Markdown"
-              * Automatically mint DOIs and BiBTeX entries
-              * Link your ORCID to your GCN Circulars
+            ## Modernization of GCN Circulars (Transition April 17, 2023)
+            * Browse and search Circulars in our all-new archive.
+            * Sign up for and manage your own email subscriptions.
+            * Enroll yourself and your colleagues to submit Circulars with arXiv-style
+              peer endorsements for new contributors.
+            * Submit Circulars with our new Web form, or continue to submit by email.
           </textarea>
         </div>
       </div>
     </section>
-    
+
+    <section data-markdown>
+      <textarea data-template>
+        ## Here is what you need to know about preparing for the transition on April 17.
+
+        * If you use Circulars now, we will transfer your settings automatically.
+        * If you receive Circulars now, you will continue to receive them.
+        * If you submit Circulars now, you will still be able to submit from
+          the same email addresses.
+        * Emails from Circulars will come from a new address, no-reply@gcn.nasa.gov.
+        * We will encourage you to submit Circulars to the new address,
+          circulars@gcn.nasa.gov, but we will still support the old address
+          gcncirc@capella2.gsfc.nasa.gov.
+        * The new archive, https://gcn.nasa.gov/circulars, will include all past
+          Circulars. We will freeze the old archive,
+          https://gcn.gsfc.nasa.gov/gcn3_archive.html.
+        
+        </textarea>
+      </section>
+
+      <section>
+          <div class="image-and-caption">
+            <div>    
+              <img src="img/screen-shot-gcn-markdown.png" width="700" alt="Screen shot of GCN Circular web form">
+            </div>
+            <div data-markdown>
+              <textarea data-template>
+              ## Planned enhancements to GCN Circulars
+              * "Astro-Flavored Markdown"
+              * Automatically mint DOIs and BiBTeX entries
```

Comment:
It isn't fully automatic and we haven't discussed anything with ADS later.

Reply:
It's automatic for the old archive.  We need to follow-up with ADS to update this soon.  They are not minting DOIs, and none of it happens quickly.